product/reports/670_Performance by Asset Type - Middleware Servers/170_Web Metrics - every minute over last hour.yaml

```diff
@@ -0,0 +1,59 @@
+---
+where_clause:
+dims:
+reserved:
+title: "EAP Web Metrics - every minute over the last hour"
+conditions:
+order: Descending
+graph:
+menu_name: "EAP Web Metrics - every minute over the last hour"
+rpt_group: Custom
+priority:
+col_order:
+- id
+- name
+- start_time
+- mw_aggregated_active_web_sessions_max
+- mw_aggregated_max_active_web_sessions_max
+- mw_aggregated_expired_web_sessions_max
+- mw_aggregated_rejected_web_sessions_max
+- mw_agregated_servlet_time_avg
```

Comment:
mw_aggregated_servlet_time_avg , there is only one 'g'

Reply:
I have to add that there is a bug here. In the hawkular agent, is it written with one 'g'.
Check it out here: [ManageIQ/manageiq/product/live_metrics/middleware_server.yaml](https://github.com/ManageIQ/manageiq/blob/master/product/live_metrics/middleware_server.yaml#L21)

Reply:
@abonas how should I proceed to open a discussion on bug?